mne/io/nihon/nihon.py

```diff
@@ -284,9 +295,15 @@ def _read_nihon_annotations(fname):
                     warn(f'Could not decode log as one of {_encodings}')
                     continue
                 t_desc = t_log[:20].strip('\x00')
+                if version == _valid_headers[-1]:
+                    t_sub_desc = t_sub_log[:20].strip('\x00')
+                    t_desc = t_desc + t_sub_desc
```

Comment:
Does it make sense to concatentate these with no separator? I would have expected maybe `' '.join([t_desc, t_sub_desc])` or `f'{t_desc} - {t_sub_desc}'` or something similar, but that's just a guess based on the variable names.

Reply:
From my results, it does seem appropriate to concatenate with no separator.